data/docs/userguide/collect_systemd_logs.mdx

```diff
@@ -0,0 +1,403 @@
+
+##### Install SigNoz
+
+Follow [this guide](https://signoz.io/docs/install/docker/) to install self-hosted SigNoz.
+
+##### Use OpenTelemetry Collector Contrib
+
+Instead of the standard collector, you need to use the contrib version that includes the JournalD receiver. 
+
+Modify the `docker-compose.yaml` file in the `deploy/docker` directory to use the contrib collector:
+
+```yaml
+services:
+  otel-collector:
+    image: otel/opentelemetry-collector-contrib:0.116.0  # Use contrib version
+    command: ["--config=/etc/otelcol-contrib/config.yaml"]
+    volumes:
+      - ./otel-collector-config.yaml:/etc/otelcol-contrib/config.yaml
+      - /var/log/journal:/var/log/journal:ro  # Mount journal directory
+    depends_on:
+      - clickhouse
+    restart: on-failure
+    user: "0:0"  # Run as root to access journal
+```
+
+##### Configure JournalD Receiver
+
+Update `otel-collector-config.yaml` with the following configuration:
+
+```yaml
+receivers:
+  journald:
+    directory: /var/log/journal
+    start_at: end
+    priority: info
+
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+      http:
+        endpoint: 0.0.0.0:4318
+
+processors:
+  batch:
+    send_batch_size: 10000
+    timeout: 10s
+
+exporters:
+  clickhouselogsexporter:
+    dsn: tcp://clickhouse:9000/signoz_logs
+    timeout: 5s
+    sending_queue:
+      queue_size: 100
+    retry_on_failure:
+      enabled: true
+      initial_interval: 5s
+      max_interval: 30s
+      max_elapsed_time: 300s
+
+service:
+  pipelines:
+    logs:
+      receivers: [otlp, journald]
+      processors: [batch]
+      exporters: [clickhouselogsexporter]
+```
+
+<Admonition type="warning">
+Note: Running the collector as root (`user: "0:0"`) is required to access journal files. In production, consider creating a dedicated user with journal access permissions.
+</Admonition>
+
+##### Update Pipeline Configuration
+
+The pipeline is already configured above to include the journald receiver.
+
+Restart the collector for the changes to take effect:
+```bash
+docker-compose restart otel-collector
+```
+
+#### Scenario 2: SigNoz on a Different Host
+
+##### Create OTel Collector Configuration
+
+Create an `otel-collector-config.yaml` file:
+
+```yaml
+receivers:
+  journald:
+    directory: /var/log/journal
+    start_at: end
+    units:  # Optional: filter specific services
+      - "nginx.service"
+      - "myapp.service"
+    priority: info  # Optional: filter by priority
+
+processors:
+  batch:
+    send_batch_size: 10000
+    send_batch_max_size: 11000
+    timeout: 10s
+
+exporters:
+  otlp/log:
+    endpoint: http://<signoz-host>:<port>
+    tls:
+      insecure: true
+
+service:
+  pipelines:
+    logs:
+      receivers: [journald]
+      processors: [batch]
+      exporters: [otlp/log]
+```
+
+<Admonition type="info">
+Replace `<signoz-host>` and `<port>` with your SigNoz instance details. For SigNoz Cloud, use the provided OTLP endpoint.
```

Comment:
This is in the Self-host section. Remove the line related to SigNoz Cloud: `For SigNoz Cloud, use the provided OTLP endpoint.`